### PR TITLE
The system keeps previously got titles in region even after users joining to another city or region

### PR DESCRIPTION
**Pull request by Bodasyuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by OleksandrBilan — APPROVED** [review]

**Review by nicoolle — APPROVED** [review]

**Review by christykobyletska — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs [hunk 1]
```diff
@@ -116,8 +119,22 @@ public async Task<CityMembersDTO> AddFollowerAsync(int cityId, string userId)
                 User = await _userManager.FindByIdAsync(userId)
                 
             };
-
             await _repositoryWrapper.CityMembers.CreateAsync(cityMember);
+            var regId = _cityService.GetByIdAsync(cityId).Result.RegionId;
```
Comment (sharix4) [inline EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:123]: await _cityService If I'm not mistaken

**Review by sharix4 — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs (hunk 1 above)
Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:123]: image

**Review by petromonka — APPROVED** [review]

**Review by sharix4 — CHANGES_REQUESTED** [review]

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs]: changed

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs [hunk 2]
```diff
@@ -116,8 +119,22 @@ public async Task<CityMembersDTO> AddFollowerAsync(int cityId, string userId)
                 User = await _userManager.FindByIdAsync(userId)
                 
             };
-
             await _repositoryWrapper.CityMembers.CreateAsync(cityMember);
+            var regId = await _cityService.GetByIdAsync(cityId);
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:123]: Is it `regId` supposed to be `regionId`?
If yes, then I guess my option quite better

File: EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs [hunk 3]
```diff
@@ -160,18 +160,20 @@ public async Task DeleteAdminByIdAsync(int Id)
 
         public async Task<IEnumerable<RegionAdministrationDTO>> GetUsersAdministrations(string userId)
         {
+
```
Comment (ostappppppp) [inline EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs:163]: Do you need here this empty space?

File: EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs [hunk 4]
```diff
@@ -296,14 +310,27 @@ public async Task AddFollowerAsyncWithUser_Valid_Test()
...
+                .ReturnsAsync(cityDto); ;
+            _repoWrapper
+                .Setup(x => x.RegionAdministration.GetAllAsync(It.IsAny<Expression<Func<RegionAdministration, bool>>>(),
+                    It.IsAny<Func<IQueryable<DataAccess.Entities.RegionAdministration>, IIncludableQueryable<DataAccess.Entities.RegionAdministration, object>>>()))
+                .ReturnsAsync(value:new List<RegionAdministration>());
+            _repoWrapper
+                .Setup(x => x.RegionAdministration.Update(It.IsAny<RegionAdministration>()));
             // Act
-            var result = await _cityParticipantsService.AddFollowerAsync(It.IsAny<int>(), It.IsAny<User>());
+            var result = await _cityParticipantsService.AddFollowerAsync(It.IsAny<int>(), "1");
```
Comment (ostappppppp) [inline EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs:324]: Please, avoid using magic numbers

File: EPlast/EPlast.WebApi/Controllers/UserController.cs [hunk 5]
```diff
@@ -150,6 +150,7 @@ await _userService.IsUserInRegionAsync(currentUser, focusUser)
                     User = _mapper.Map<UserDTO, UserViewModel>(focusUser),
                     TimeToJoinPlast = (int)time.TotalDays,
                     IsUserPlastun = isFocusUserPlastun,
+
```
Comment (ostappppppp) [inline EPlast/EPlast.WebApi/Controllers/UserController.cs:153]: also empty line here

**Review by sharix4 — APPROVED** [review]

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs (hunk 2 above)
Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:123]: Ok, thanks

File: EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs (hunk 3 above)
Comment (Bodasyuk) [reply EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs:163]: No, thanks

File: EPlast/EPlast.WebApi/Controllers/UserController.cs (hunk 5 above)
Comment (Bodasyuk) [reply EPlast/EPlast.WebApi/Controllers/UserController.cs:153]: thanks

File: EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs (hunk 4 above)
Comment (Bodasyuk) [reply EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs:324]: Ok, thaks

**Pull request merged**